cluster-up/cluster/okd-4.1.0-nodes3/provider.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+image="okd-4.1.0@sha256:5ff43c79338d19e550a9ca8ef67f09d82a3425590970607888249cb3ccae6ea8"
+
+source ${KUBEVIRTCI_PATH}/cluster/ephemeral-provider-common.sh
+
+function _port() {
+    ${_cli} ports --prefix $provider_prefix --container-name cluster "$@"
+}
+
+function up() {
+    params="--random-ports --background --prefix $provider_prefix --master-cpu 5 --workers-cpu 5 --registry-volume $(_registry_volume) kubevirtci/${image}"
```

Comment:
I believe you need to specify the number of nodes here, but anyway okd provider still does not support it, so I prefer to drop this provider until we will have a working solution to configure additional nodes under the OKD, in general, it should not be too hard, it only need to scale worker machineset.

Reply:
yep, i'll drop it for now. We can have this discussion later.